### DISCUSSION TITLE
How to handle blockdata for whitelist?

### DISCUSSION THREAD

**Discussion by lerokko** [jbredwards/Fluidlogged-API#135] [Q&A]
I see the whitelist supports metadata but that is only the static numeric one.
Industrialcraft for example has 1 block called "ic:te" that handles almost everything, most machines (full blocks) but also the cables.
There are block tags specifying that it is a cable and not a machine as well as what cable it is.A lot of other mods do this as well.
Is it possible to allow only the cables of this mod to be waterlogged without everything else? Since the do not simply use metadata but rather blockstates.

**Comment by jbredwards** [comment]
This mod's config does not support fluidlogging based on tile entity tags. One way for you to add this would be use a mod like [GroovyScript](https://groovyscript-docs.readthedocs.io/en/latest/groovyscript/minecraft/events/) to add an event listener for [`FluidloggableEvent`](https://github.com/jbredwards/Fluidlogged-API/blob/1.12.2-v1.7-Rewrite/src/main/java/git/jbredwards/fluidlogged_api/api/event/FluidloggableEvent.java) and set its result to `Event.Result.ALLOW` when event.state is an "ic:te" & when the tile entity at the pos has the desired block tag.
